ui-components/src/global/forms.scss

```diff
@@ -1,4 +1,10 @@
 .field {
+  --bordered-border-width: 1px;
+  --bordered-border-radius: var(--bloom-s1_5);
+  --bordered-vertical-padding: var(--bloom-s4);
+  --bordered-leftward-padding: 2.875rem;
+  --bordered-checked-bg-color: var(--bloom-color-gray-100);
+
```

Comment:
I recognize that the naming here is a bit awkward, but the "bordered" prefix seemed necessary for theming variables that are only applied to a specific variation of the component. Likely, will change with the refactor if radio + checkbox are separated out.